app/models/identified_pathways_assessment.rb

```diff
@@ -0,0 +1,28 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/master/LICENSE.md
+###
+
+class IdentifiedPathwaysAssessment < IdentifiedClientAssessment
+  attr_accessor :youth_rrh_aggregate, :dv_rrh_aggregate
+  after_find :populate_aggregates
+
+  private def populate_aggregates
+    if youth_rrh_desired? && rrh_desired?
+      self.youth_rrh_aggregate = 'both'
+    elsif youth_rrh_desired?
+      self.youth_rrh_aggregate = 'youth'
+    elsif rrh_desired?
+      self.youth_rrh_aggregate = 'adult'
+    end
+
+    if dv_rrh_desired? && rrh_desired?
+      self.dv_rrh_aggregate = 'both'
+    elsif dv_rrh_desired?
+      self.dv_rrh_aggregate = 'dv'
+    elsif rrh_desired?
+      self.dv_rrh_aggregate = 'non-dv'
+    end
```

Comment:
Is there a difference here between identified and de-identified?  If not, let's move it to `IdentifiedClientAssessment`